### PR TITLE
fix(mrf): design review add workflow page

### PR DESCRIPTION
**Pull request by KenLSM**
## Problem

Closes FRM-1631

Add workflow page
- Change empty state header copy to - ‘Create a workflow to collect responses from multiple respondents in the same form submission’
- Change empty state copy to - ‘Assign respondents to specific steps, and control which fields they can fill. Learn how to create a workflow.’
- Infobox when creating workflow should say - ‘Add multiple steps, and for each step, assign a respondent, and select fields that they can fill.’
- Change to Fields to fill instead of questions to fill
- Change icon beside fields to fill to the outline version
- Change icon copy to ‘Respondent will only be able to fill the fields you have selected’

**Breaking Changes** 

- No - this PR is backwards compatible  

## Before & After Screenshots

| Component | Before | After |
|--------|--------|--------|
| Empty Workflow | Screenshot 2024-03-28 at 4 59 03 PM | Screenshot 2024-03-28 at 5 00 13 PM |
| Workflow Step Block | Screenshot 2024-03-28 at 5 00 21 PM | Screenshot 2024-03-28 at 6 27 06 PM |

### CODE REVIEW

File: frontend/src/features/admin-form/create/workflow/components/EmptyWorkflow.tsx [hunk 1]
```diff
@@ -24,11 +24,12 @@ export const EmptyWorkflow = (): JSX.Element => {
...
     >
       <Text textStyle="h2" as="h2">
-        Create a workflow to collect responses from multiple respondents
+        Create a workflow to collect responses from multiple respondents in the
+        same form submission
       </Text>
       <Text textStyle="body-1" mt="1rem">
         Assign respondents to specific steps, and control which fields they can
-        see and fill.{' '}
+        fill. Learn how to create a workflow.{' '}
```
Comment (kathleenkhy) [inline frontend/src/features/admin-form/create/workflow/components/EmptyWorkflow.tsx:32]: Does it make sense for "Learn how to create a workflow" to be repeated twice? Should the first "Learn how to create a workflow" be removed?

File: frontend/src/features/admin-form/create/workflow/components/EmptyWorkflow.tsx [hunk 2]
```diff
@@ -24,11 +24,12 @@ export const EmptyWorkflow = (): JSX.Element => {
...
         <Link isExternal href={GUIDE_FORM_MRF}>
           Learn how to create a workflow
```
Comment (kathleenkhy) [inline frontend/src/features/admin-form/create/workflow/components/EmptyWorkflow.tsx:34]: Does it make sense for "Learn how to create a workflow" to be repeated? Should the first "Learn how to create a workflow" be removed?

Comment (KenLSM) [reply frontend/src/features/admin-form/create/workflow/components/EmptyWorkflow.tsx:34]: Good catch. Typo-ed here!

**Review by kathleenkhy — APPROVED** [review]
lgtm

**Pull request merged**